unittests/contracts/read_only_query_tests.abi

```diff
@@ -0,0 +1,105 @@
+{
```

Comment:
I think it would also be a good idea to add the smart contract(cpp file) used to generate this abi and wasm.

Reply:
The source file for the test contract is part of `eosio.cdt`.  We've decided to avoid versioning problems likely to arise from keeping two copies of it.

Reply:
got it, i think the eosio.cdt is https://github.com/EOSIO/eosio.cdt/pull/1068, let me know if it is not